certified-connectors/ID Analyzer/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2871 @@
+              "type": "object",
+              "properties": {
+                "success": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "success"
+                },
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "createtime": {
+                      "type": "string",
+                      "description": "createtime"
+                    },
+                    "updatetime": {
+                      "type": "string",
+                      "description": "updatetime"
+                    },
+                    "trustlevel": {
+                      "type": "string",
+                      "description": "trustlevel"
+                    },
+                    "trustnote": {
+                      "type": "string",
+                      "description": "trustnote"
+                    },
+                    "docupass_reference": {
+                      "type": "string",
+                      "description": "docupass_reference"
+                    },
+                    "docupass_success": {
+                      "type": "string",
+                      "description": "docupass_success"
+                    },
+                    "docupass_failedreason": {
+                      "type": "string",
+                      "description": "docupass_failedreason"
+                    },
+                    "docupass_customid": {
+                      "type": "string",
+                      "description": "docupass_customid"
+                    },
+                    "documentNumber": {
+                      "type": "string",
+                      "description": "documentNumber"
+                    },
+                    "documentNumber_formatted": {
+                      "type": "string",
+                      "description": "documentNumber_formatted"
+                    },
+                    "personalNumber": {
+                      "type": "string",
+                      "description": "personalNumber"
+                    },
+                    "firstName": {
+                      "type": "string",
+                      "description": "firstName"
+                    },
+                    "middleName": {
+                      "type": "string",
+                      "description": "middleName"
+                    },
+                    "lastName": {
+                      "type": "string",
+                      "description": "lastName"
+                    },
+                    "fullName": {
+                      "type": "string",
+                      "description": "fullName"
+                    },
+                    "firstName_local": {
+                      "type": "string",
+                      "description": "firstName_local"

```

Comment:
@ccyy915 Can you please provide valid description values as the descriptions are visible to public. And the given values are looking like IDs.

Applies to all the descriptions in the swagger